staging/src/k8s.io/apiserver/pkg/util/logs/logs.go

```diff
@@ -67,3 +68,12 @@ func FlushLogs() {
 func NewLogger(prefix string) *log.Logger {
 	return log.New(GlogWriter{}, prefix, 0)
 }
+
+// GlogSetter is a setter to set glog level.
+func GlogSetter(val string) (string, error) {
+	var level glog.Level
+	if err := level.Set(val); err != nil {
+		return "", fmt.Errorf("failed set glog.logging.verbosity %s: %v", val, err)
+	}
+	return "successfully set glog.logging.verbosity to " + val, nil
```

Comment:
nit: prefer
```
fmt.Sprintf("successfully set glog.logging.verbosity to %d", level)
```

Reply:
Actually, `level.Set(val)` weirdly doesn't actually modify the level value, so nevermind...

Reply:
ok

Reply:
`fmt.Sprintf("successfully set glog.logging.verbosity to %s", val)`